plugins/transforms/ldap/src/main/java/org/apache/hop/pipeline/transforms/ldapinput/LdapConnection.java

```diff
@@ -249,21 +249,22 @@ public void search( String searchBase, String filter, int limitRows, String[] at
       if ( isPagingUsed() ) {
         // paging is activated
         // Request the paged results control
-        ctlp = new PagedResultsControl( GetPagingSize(), Control.CRITICAL );
+        ctlp = new PagedResultsControl( getPagingSize(), Control.CRITICAL );
         nrCtl++;
         if ( log.isDebug() ) {
-          log.logDebug( BaseMessages.getString( "LdapInput.Log.PageSize", String.valueOf( GetPagingSize() ) ) );
+          log.logDebug( BaseMessages.getString( "LdapInput.Log.PageSize", String.valueOf( getPagingSize() ) ) );
         }
       }
 
       if ( nrCtl > 0 ) {
         Control[] ctls = new Control[ nrCtl ];
         int index = 0;
         if ( ctlk != null ) {
-          ctls[ index++ ] = ctlk;
+          ctls[ index ] = ctlk;
+          index++;
         }
         if ( ctlp != null ) {
-          ctls[ index++ ] = ctlp;
+          ctls[ index ] = ctlp;
```

Comment:
Missing index++

Reply:
It was unnecessary increment.
https://sonarcloud.io/project/issues?id=hop&issues=AXG4QflqIq6iCIhl5L7h&open=AXG4QflqIq6iCIhl5L7h